modules/stanford_soe_helper_homepage/stanford_soe_helper_homepage.install

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ */
+
+
+/**
+ * Implements hook_install().
+ */
+function stanford_soe_helper_homepage_install() {
+
+  $context_status = variable_get('context_status', array());
+  $context_status['stanford_jumpstart_soe_home'] = FALSE;
+  $context_status['stanford_jumpstart_home_terman'] = TRUE;
+  variable_set('context_status', $context_status);
+
+  $cbc_layouts = variable_get('contextual_block_class', array());
+  $cbc_layouts['stanford_jumpstart_soe_home']['views-stanford_events_views-block'][] = 'span4 well';
+  $cbc_layouts['stanford_jumpstart_soe_home']['bean-homepage-about-block'][] = 'span4 well';
+  $cbc_layouts['stanford_jumpstart_soe_home']['views-9bf4ec9695a5b13242ba5a4898a6b635'][] = 'span4 well';
```

Comment:
Please add a comment about which view and display this is. It is hard to track down.